docs/release/cut-sw-release.md

```diff
@@ -368,36 +372,38 @@ cd docker-osg-wn
 
-1.  The release manager writes the release announcement and send it out. Here is a sample, replace `<BRACKETED TEXT>` with the appropriate values:
-
-         Subject: Announcing OSG Software version <VERSION(S)>
-         
-         We are pleased to announce OSG Software version <VERSION(S)>!
-         
-         This is the OSG Software distributed via RPMs for:
-         
-         * Scientific Linux 6 and 7
-         * CentOS 6 and 7
-         * Red Hat Enterprise Linux 6 and 7
-         
-         This release affects the <SET OF METAPACKAGES (client, compute element, etc...)>. Changes include:
-         
-         * Major change 1
-         * Major change 2
-         * Major change 3
-         
-         Release notes and pointers to more documentation can be found at:
-         
-         <LINK TO RELEASE NOTES>
-         
-         Need help? Let us know:
-         
-         https://opensciencegrid.github.io/docs/common/help/
-         
-         We welcome feedback on this release
+1.  The release manager writes the a release announcement for each version and sends it out.
+    The announcement should mention a handful of the most important updates.
+    Due to downstream formatting issues, each major change should end at column 76 or earlier.
+    Here is a sample, replace `<BRACKETED TEXT>` with the appropriate values:
+
+        Subject: Announcing OSG Software version <VERSION>
+
+        We are pleased to announce OSG Software version <VERSION>!
+
+        Changes to OSG <VERSION> include:
+        - Major Change 1
+        - Major Change 2
+        - Major Change 3
+
+        Release notes and pointers to more documentation can be found at:
+
+        http://opensciencegrid.github.io/docs/release/<SERIES.VERSION>/release-<RELEASE-VERSION>/
+
+        Need help? Let us know:
+
+        http://opensciencegrid.github.io/docs/common/help/
+
+        We welcome feedback on this release!
 
 2.  The release manager emails the announcement to `vdt-discuss@opensciencegrid.org`
 3.  The release manager asks the GOC to distribute the announcement by [opening a ticket](https://ticket.opensciencegrid.org/goc/other)
 4.  The release manager closes the tickets marked 'Ready for Release' in the release's JIRA filter using the 'bulk change' function.
     Also set the Fix Versions field to the appropriate value(s) and uncheck the box that reads "Send mail for this update"
 
+Day 3: Update the ITB
+---------------------
+
+Now that the release has had a chance to propogate to all the mirrors, update the Madison ITB site by following
+the [yum update section](../infrastructure/madison-itb/#doing-yum-updates) of the Madison ITB document.
+Remember, it may be advisable to stop the HTCondor daemons according to the [HTCondor pre-release testing instructions](itb-testing/#installing-htcondor-prerelease).
```

Comment:
Let's be clearer here; tell them when they "may" want to stop the condor daemons and which hosts they need to stop them on. Or you can just tell them to do it every time.